profile/CSIP.xml

```diff
@@ -1224,7 +1224,8 @@
             <requirement ID="CSIP105" REQLEVEL="SHOULD" EXAMPLES="structMapExample2">
                 <description>
                     <head>Representation divisions</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">When the transfer consist of representations there are one representation div present for each representation</p>
+                    <p xmlns="http://www.w3.org/1999/xhtml">When the transfer consist of content divded into representations described by its own METS.xml documents there are one representation div present for each representation.</p>
```

Comment:
When a package consists of multiple representations, each described by a representation level METS.xml document, there is a discrete representation div element for each representation.